
#x - The x coordinate of the lower left corner of a rectangle
#y - The y coordinate of the lower left corner of a rectangle
#width - The width of the rectangle
#height - The height of the rectangle
#trans_vel - The translational velocity of the robot
#rot_vel - The rotational velocity of the robot
#frequency - The frequency of the control loop

x: 0
y: 0
width: 1
height: 1
trans_vel: 0.5
rot_vel: 1
frequency: 100
